config/config.go

```diff
@@ -205,11 +205,11 @@ type Log struct {
 	// File log config.
 	File logutil.FileLogConfig `toml:"file" json:"file"`
 
+	EnableSlowLog       bool   `toml:"enable-slow-log" json:"enable-slow-log"`
```

Comment:
How about using `nullableBool`?

Reply:
If `enable-slow-log` is unset, it defaults to `true`. Thus it's not necessary to use `nullableBool.`